modules/executors/variables.tf

```diff
@@ -91,16 +91,16 @@ variable "max_active_time" {
   description = "The maximum time that can be spent by the worker dequeueing records to be handled"
 }
 
-variable "firecracker_num_cpus" {
+variable "job_num_cpus" {
```

Comment:
should we do something here to stay backwards compatible? Ie. keep the variable name as well and do a ternary at the usage of the variable?